packages/fhir-converter/src/templates/cda/Sections/Assessments.hbs

```diff
@@ -0,0 +1,11 @@
+{{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.8')}}
+    {{#with (getFirstCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.8')}}
+        {{#each (toArray 2_16_840_1_113883_10_20_22_2_8) as |assessment|}}
+            {{#if assessment.text}}
+                {{#with (extractAndMapTableData assessment.text) as |assessmentText|}}
+                    {{>References/DiagnosticReport/assessment.hbs diagReport=assessment text=assessmentText ID=(generateUUID (toJsonString assessment))}},
+                {{/with}}
+            {{/if}}
+        {{/each}}
+    {{/with}}
+{{/if}}
```

Comment:
let's leave a big fat TODO here so someone else doesn't come along and use it - until we come up with a good place to put these